### ISSUE TITLE
KeyError: 'children'

### ISSUE THREAD

**Issue by andreitulpan** [ParthJadhav/Tkinter-Designer#62]
Hi! I have a problem when i click generate. The error is: KeyError: 'children'
I attached a photo with the error on imgur: [https://i.imgur.com/2Ni8JrM_d.webp?maxwidth=760](url)
My figma: https://www.figma.com/file/1HMX8V1VpfbCH2rava1wPs/Untitled?node-id=0%3A1

**Comment by andreitulpan** [comment]
I have a new error: [https://i.imgur.com/yZceAoJ.jpg](url)

**Comment by ParthJadhav** [comment]
Thanks for the information! Let me see!

**Comment by ParthJadhav** [comment]
Can you type the new error... I can't open that link.

**Comment by andreitulpan** [comment]
Is too long, you can copy the link is an imgur link.

**Comment by andreitulpan** [comment]
It works now but the app is not what i did in Figma. The code generated is this:
`from tkinter import *


def btn_clicked():
    print("Button Clicked")


window = Tk()

window.geometry("750x500")
window.configure(bg = "#FFFFFF")
canvas = Canvas(
    window,
    bg = "#FFFFFF",
    height = 500,
    width = 750,
    bd = 0,
    highlightthickness = 0,
    relief = "ridge")
canvas.place(x = 0, y = 0)

window.resizable(False, False)
window.mainloop()`
And you saw what i did in Figma.

**Comment by ParthJadhav** [comment]
Okay, Let me try again

**Comment by ParthJadhav** [comment]
Yes, So I think the problem is in naming. Can you check the naming guide in instructions and name your elements according to the instructions...

**Comment by andreitulpan** [comment]
I did this, i receive this message in terminal: list index out of range.

**Comment by ParthJadhav** [comment]
I didn't see the naming of your Figma document cause I don't have access to my PC. It would be helpful if you send the screenshot of layers pannel on the left.

**Comment by ParthJadhav** [comment]
Also directly upload it on GitHub, Imgur is not working

**Comment by andreitulpan** [comment]
![image](https://user-images.githubusercontent.com/72267834/124306230-cb61c580-db6e-11eb-9cfe-be1d86ee2259.png)
![image](https://user-images.githubusercontent.com/72267834/124306272-d87eb480-db6e-11eb-9664-1d80e5c0dd16.png)
I didn't know that i can upload photos there. I don't know what i named wrong.

**Comment by ParthJadhav** [comment]
Can you remove the rectangles out of the rectangle group and keep all the rectangles separate...

**Comment by andreitulpan** [comment]
Now i receive the error with children.

**Comment by ParthJadhav** [comment]
Can you share the updated layers pannel?

**Comment by andreitulpan** [comment]
Sure

**Comment by andreitulpan** [comment]
![image](https://user-images.githubusercontent.com/72267834/124306820-9ace5b80-db6f-11eb-9024-7086c0545a48.png)

**Comment by ParthJadhav** [comment]
Ohh, I see. Can you give me editing permission ?

**Comment by andreitulpan** [comment]
Yes

**Comment by andreitulpan** [comment]
Done

**Comment by ParthJadhav** [comment]
Try now

**Comment by andreitulpan** [comment]
![image](https://user-images.githubusercontent.com/72267834/124307195-234cfc00-db70-11eb-93d5-031c5a2db58f.png)
It works but the result is kind of different.

**Comment by andreitulpan** [comment]
How you did the window layer?

**Comment by ParthJadhav** [comment]
Many things -

1. Font size is greater in Linux and Windows (You can reduce it manually from window.py)
2. If you want to have text on diffrent lines then press enter key. It wont work if it automatically moves the cursor to the next line.
3. The window thing is a Frame. It's similar to the Tkinter window in which all the elements sit. It's called Frame in Figma. Press F or click here.
<img width="48" alt="Screenshot 2021-07-02 at 10 33 39 PM" src="https://user-images.githubusercontent.com/42001064/124307517-8eed9400-db85-11eb-8acf-8d5c78a2748d.png">

I'll fix the font issues for you....

**Comment by ParthJadhav** [comment]
Try now ...

**Comment by ParthJadhav** [comment]
If it's working you can remove me from editing permissions.

**Comment by andreitulpan** [comment]
It works ok. Thanks!
